NOTICE

```diff
@@ -55,6 +58,7 @@ This software includes third party software subject to the following copyrights:
 - libsodium - Copyright (c) 2013 - 2018 Frank Denis under the ISC software license
 - IANA timezone database - public domain
 - date (HowardHinnant/date) - notices below
+- JSON for Modern C++ (nlohmann/json) - Copyright (c) 2013-2021 Niels Lohmann
```

Comment:
Is this a hard dependency of the azure sdk? Because we already have `rapidjson` for json handling, so if it's possible to avoid bringing in another lib for the same purpose, I would try.

Reply:
In the latest version it has become a new dependency for azure sdk and does not compile without it. We may investigate if it's viable to patch and replace it with rapidjson in the sdk, although it may be cumbersome to maintain it in the future.

Reply:
Ok, don't bother